.github/workflows/workflow.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
     steps:
       - id: set-matrix
         run: |
-          echo "::set-output name=matrix::{\"node-version\": [\"$NODE_LTS_MAINTENANCE_VERSION\", \"$NODE_LTS_ACTIVE_VERSION\"]}"
+          echo "matrix={\"node-version\": [\"$NODE_LTS_MAINTENANCE_VERSION\", \"$NODE_LTS_ACTIVE_VERSION\"]}" >> $GITHUB_OUTPUT
```

Comment:
Is it possible to set up the matrix of Nodejs versions normally (with an array of numbers) instead of constructing it like this? This seems rather "hacky".

Reply:
Well, I tried a bunch of thing, seems that this is the way to go if we don't want to repeat the matrix over and over for each jobs